core/src/test/java/org/mobilitydata/gtfsvalidator/input/GtfsInputTest.java

```diff
@@ -66,4 +69,24 @@ public void zipInput() throws IOException {
         GtfsInput gtfsInput = GtfsInput.createFromPath(zipFile.getAbsolutePath());
         assertThat(gtfsInput.getFilenames()).containsExactly("stops.txt");
     }
+
+    @Test
+    public void createFromValidUrlShouldNotThrowException() throws IOException, URISyntaxException, InterruptedException {
+        GtfsInput underTest = GtfsInput.createFromUrl(
+                new URL("https://octa.net/current/google_transit.zip"),
+                "storage");
+        assertThat(underTest instanceof GtfsZipFileInput);
+        // remove created file
+        File toDelete = new File("storage");
+        assertTrue(toDelete.delete());
+
+    }
+
+    @Test
+    public void createFromInvalidUrlShouldThrowException() {
+        assertThrows(
+                IOException.class, () -> GtfsInput.createFromUrl(
+                        new URL("https://octa.net/current/gowkdjhfiouwehfogle_transit.zip"),
```

Comment:
good point, care to ellaborate a little? :)

Reply:
>good point, care to ellaborate a little? :)

Generally you want to use URLs under your control in tests so they don't break randomly if the owner of the URL decides to change something. Granted, in this case it's unlikely that OCTA decides to publish a valid GTFS named `gowkdjhfiouwehfogle_transit.zip` that would break this particular test, but for consistency with other tests that depend on live URLs IMHO it would be better if they are all under domains that we have control of.